auto_nag/tests/test_query_creator.py

```diff
@@ -0,0 +1,66 @@
+
+try:
+    # If relman-auto-nag is installed
+    from auto_nag.bugzilla.utils import os
+    from auto_nag.scripts.query_creator import (getTemplateValue, getReportURL,
+                                                createQueriesList)
+    import shutil
+except:
+    # If relman-auto-nag not installed, add project root directory into
+    # PYTHONPATH
+    import os
+    import sys
+    import inspect
+    import shutil
+    currentdir = os.path.dirname(os.path.abspath(
+                                 inspect.getfile(inspect.currentframe())))
+    parentdir = os.path.dirname(currentdir)
+    sys.path.insert(0, parentdir)
+    from auto_nag.scripts.query_creator import (getTemplateValue, getReportURL,
+                                                createQueriesList)
+
+class TestQueryCreator:
+    def test_getTemplateValue(self):
+        """
+        Expecting a VERSION number
+        """
+        url = "https://wiki.mozilla.org/Template:BETA_VERSION"
+        beta_version = getTemplateValue(url)
+        assert type(int(beta_version)) is not type(int)
+
+    def test_getReportURL(self):
+        """
+        Expectong proper URL with Bug numbers
```

Comment:
Anu is writing test cases for utils, so she told me she will write today
itself if my pull got merged

On Mon, Oct 26, 2015, 1:04 PM Sylvestre Ledru <notifications@github.com>
wrote:

> In auto_nag/tests/test_query_creator.py
> <https://github.com/mozilla/relman-auto-nag/pull/38#discussion_r42965391>:
>
> > +    sys.path.insert(0, parentdir)
> > +    from auto_nag.scripts.query_creator import (getTemplateValue, getReportURL,
> > +                                                createQueriesList)
> > +
> > +class TestQueryCreator:
> > +    def test_getTemplateValue(self):
> > +        """
> > +        Expecting a VERSION number
> > +        """
> > +        url = "https://wiki.mozilla.org/Template:BETA_VERSION"
> > +        beta_version = getTemplateValue(url)
> > +        assert type(int(beta_version)) is not type(int)
> > +
> > +    def test_getReportURL(self):
> > +        """
> > +        Expectong proper URL with Bug numbers
>
> expecting I guess ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/relman-auto-nag/pull/38/files#r42965391>.
>
-- 

~anoop_valluthadam